MADRaS/envs/gym_madras.py

```diff
@@ -13,30 +13,36 @@
 """
 
 import math
-import gym
 from copy import deepcopy
 import numpy as np
 import MADRaS.utils.snakeoil3_gym as snakeoil3
 from MADRaS.utils.gym_torcs import TorcsEnv
 from MADRaS.controllers.pid import PID
-
-
-
-class MadrasEnv(TorcsEnv, gym.Env):
+import gym
+import os
+import subprocess
+import signal
+import time
+from mpi4py import MPI
+import socket
+
+class MadrasEnv(TorcsEnv,gym.Env):
     """Definition of the Gym Madras Env."""
     def __init__(self, vision=False, throttle=True,
-                 gear_change=False, port=3001, pid_assist=True,
-                 CLIENT_MAX_STEPS=np.inf):
+                 gear_change=False, port=60934, pid_assist=False,
+                 CLIENT_MAX_STEPS=np.inf,visualise=True):
         """Init Method."""
+        self.torcs_proc = None
         self.pid_assist = pid_assist
```

Comment:
I would recommend writing a comment stating the use of the newly introduced variables.